SwiftyCharms/XMLParserExamples.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  XMLParserExamples.swift
+//  SwiftyCharms
+//
+//  Created by Kyle Fang on 2/18/16.
+//  Copyright © 2016 Ruoyu Fu. All rights reserved.
+//
+
+//Example
+let input = "<hello name=\"SwiftyCharms\">world</hello>"
+
+//Seems like there are memory issue at the moment
+let input2 = "<hello name=\"SwiftyCharms\"><time name=\"SwiftyCharms\">hello</time></hello>"
```

Comment:
@lingoer this one fail to work because of memory issue... :(
How did you get around recursive reference?